src/server/module-loader.ts

```diff
@@ -167,50 +120,38 @@ export class ModuleLoader extends EventTarget {
     });
 
     await Promise.all(imports);
+
+    this.dispatchEvent(new Event("load"));
+    debug("finished loading: %s", this.basePath);
   }
 
   private async loadEndpoint(
     fullPath: string,
     directory: string,
     file: Dirent,
   ) {
-    try {
-      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
-      const endpoint: ContextModule | Module = (await this.uncachedImport(
-        fullPath,
-      )) as ContextModule | Module;
-
-      if (file.name.includes("_.context")) {
-        if (isContextModule(endpoint)) {
-          this.contextRegistry.add(
-            `/${directory.replaceAll("\\", "/")}`,
-
-            // @ts-expect-error TS says Context has no constructable signatures but that's not true?
-            // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-            new endpoint.Context(),
-          );
-        }
-      } else {
-        const url = `/${nodePath.join(
-          directory,
-          nodePath.parse(file.name).name,
-        )}`
-          .replaceAll("\\", "/")
-          .replaceAll(/\/+/gu, "/");
-
-        // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
-        this.registry.add(url, endpoint as Module);
-      }
-    } catch (error: unknown) {
-      if (
-        String(error) ===
-        "SyntaxError: Identifier 'Context' has already been declared"
-      ) {
-        // Not sure why Node throws this error. It doesn't seem to matter.
-        return;
```

Comment:
@dethell I wonder if we need to put this back in. 

Actually looking at it with fresh eyes, I bet the imported file declares `Context` twice. And what will happen if there's a syntax error in one of the files? I think I may have accidentally removed some fault tolerance with recent refactoring.